src/tools/data_handlers/analytics_handler.py

```diff
@@ -71,29 +70,61 @@ def _get_aoi_type(self, aoi: Dict) -> str:
             return "indigenous_land"
         elif aoi["subtype"] == "protected-area":
             return "protected_area"
+        # custom_area instead of custom-area because
+        # that's what the existing
```

Comment:
please ignore this comment. we should use `custom-area` and update the frontend code to do the same as well

Reply:
that's done in https://github.com/wri/project-zeno-next/pull/115